src/components/JsonSchema/JsonSchemaFormRenderer.tsx

```diff
@@ -55,12 +55,13 @@ export const JsonSchemaFormRenderer = ({
   const sharedProps = {
     id: path.join("."),
     label: labelWithSchemaType,
-    value: getNestedValue(parsedSorobanOperation.args, path.join(".")),
+    value: getNestedValue(parsedSorobanOperation.args, path.join("."))?.value,
```

Comment:
this is needed because for all inputs value, we now include `type` in addition to `value`.